src/main/java/seedu/tp/ui/Ui.java

```diff
@@ -211,6 +211,32 @@ public LocalDate promptUserForRequiredLocalDate(String fieldName) {
         return localDate;
     }
 
+    /**
+     * Gets the response from the user after the user reviewed a flashcard.
+     *
+     * @param flashcard the flashcard the user just reviewed
+     * @return the response given by the user
+     */
+    public String promptUserResponseForReviewing(Flashcard flashcard) {
+        System.out.println("Do you want to mark this flashcard as reviewed?");
+        String input = getNextLine().trim().toLowerCase();
```

Comment:
Consider using the `getNextLine()` defined in this class to get user input so that we consistently use only `getNextLine()` to get user input

Reply:
Do you mean that we have no need to create this method here since we can directly use ```getNextLine()``` in this class?

Reply:
> Do you mean that we have no need to create this method here since we can directly use `getNextLine()` in this class?

Oops, my bad. I misread your code. Please ignore this comment haha